action-scheduler.php

```diff
@@ -26,27 +26,27 @@
  * @package ActionScheduler
  */
 
-if ( ! function_exists( 'action_scheduler_register_3_dot_5_dot_0' ) && function_exists( 'add_action' ) ) {
+if ( ! function_exists( 'action_scheduler_register_3_dot_5_dot_2' ) && function_exists( 'add_action' ) ) {
 
 	if ( ! class_exists( 'ActionScheduler_Versions', false ) ) {
 		require_once __DIR__ . '/classes/ActionScheduler_Versions.php';
 		add_action( 'plugins_loaded', array( 'ActionScheduler_Versions', 'initialize_latest_version' ), 1, 0 );
 	}
 
-	add_action( 'plugins_loaded', 'action_scheduler_register_3_dot_5_dot_0', 0, 0 );
+	add_action( 'plugins_loaded', 'action_scheduler_register_3_dot_5_dot_2', 0, 0 );
 
 	/**
 	 * Registers this version of Action Scheduler.
 	 */
-	function action_scheduler_register_3_dot_5_dot_0() {
+	function action_scheduler_register_3_dot_5_dot_2() {
 		$versions = ActionScheduler_Versions::instance();
-		$versions->register( '3.5.0', 'action_scheduler_initialize_3_dot_5_dot_0' );
+		$versions->register( '3.5.0', 'action_scheduler_initialize_3_dot_5_dot_2' );
```

Comment:
I think this should be:

```suggestion
		$versions->register( '3.5.2', 'action_scheduler_initialize_3_dot_5_dot_2' );
```